crypto/cms/cms_env.c

```diff
@@ -607,6 +607,13 @@ static int cms_RecipientInfo_ktri_decrypt(CMS_ContentInfo *cms,
     if (!ossl_cms_env_asn1_ctrl(ri, 1))
         goto err;
 
+    if (EVP_PKEY_base_id(pkey) == EVP_PKEY_RSA)
```

Comment:
This should be `if (EVP_PKEY_is_a(pkey, "RSA"))`

Reply:
done